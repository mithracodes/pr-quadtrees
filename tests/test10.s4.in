144.969 -37.7965 144.9725 -37.7945
144.97 -37.7969 144.971 -37.796
144.969 -37.7969 144.97 -37.796
144.9688 -37.7964 144.9698 -37.7956
144.97039 -37.79634 144.97138 -37.7952
