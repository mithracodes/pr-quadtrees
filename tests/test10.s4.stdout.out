144.969 -37.7965 144.9725 -37.7945 --> SW SW SE
144.97 -37.7969 144.971 -37.796 --> SW SE
144.969 -37.7969 144.97 -37.796 --> SW SW SE
144.9688 -37.7964 144.9698 -37.7956 --> SW SW
144.97039 -37.79634 144.97138 -37.7952 --> SW SE
